Cargo.toml

```diff
@@ -5,13 +5,13 @@ members = [
     "async-signature",
     "cipher",
     "crypto",
-#    "crypto-common",
+    "crypto-common",
     "digest",
-    "elliptic-curve",
+#    "elliptic-curve",
     "kem",
     "password-hash",
-    "signature",
+#    "signature",
     "signature_derive",
     "universal-hash",
 ]
-exclude = ["crypto-common"]
+exclude = ["elliptic-curve", "signature"]
```

Comment:
Note: can't bump these due to their additional (dev-)dependencies which require some additional crate releases first